File: exams/andrade/unit1/Q_33/GameScreen/src/geometry/Circle.java

package geometry;

/**
 *
 * @author Alejandro Andrade, Scriptal, DCCO_ESPE
 */
public class Circle extends Shape{
    
    private double radius;
    
    public Circle(double radius){
        this.radius = radius;
    }
    
    @Override
    public double calculateArea()
    {
        return Math.PI * Math.pow(this.radius, 2);
    }
    
    @Override
    public double calculatePerimeter()
    {
        return 2 * Math.PI * this.radius;
    }

    /**
     * @return the radius
     */
    public double getRadius() {
        return radius;
    }

    /**
     * @param radius the radius to set
     */
    public void setRadius(double radius) {
        this.radius = radius;
    }
    
    
    
}

File: ws/orozco/u2/ws23/Geometry/Geometry/Circle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Geometry
{
    class Circle : IShape
    {
        public double Radius { get; set; }

        public Circle(double radius)
        {
            Radius = radius;
        }
        public double CalculateArea()
        {
            return Math.PI * Math.Pow(Radius, 2);
        }
        public double CalculatePerimeter()
        {
            return 2 * Math.PI * Radius;
        }
    }
}
